.azure/scale.csv

```diff
@@ -0,0 +1,26 @@
+stage,siteId,endingAddress,storageConnectivitySwitchless,authenticationMethod,mobile,storageNetworks,companyName,postalCode,dnsServers,servers,domainServerIP,phone,rdmaEnabled,domainFqdn,adouSuffix,location,aksArc-lnet-defaultGateway,managementAdapters,virtualHostIp,startingAddress,country,aksArc-lnet-endingAddress,rbacAdminGroupObjectIds,stateOrProvince,aksArc-lnet-dnsServers,serverPorts,aksArc-lnet-addressPrefix,contactName,defaultGateway,aksArc-lnet-startingAddress,emailList,dcPort,enableProvisioners,streetAddress1,aksArc-controlPlaneIp,city
```

Comment:
## Arc IaC Automation generated
Next steps: 
  - [ ] Setup local git hook by running command in the root folder of cloned repo: git config --local core.hooksPath ./.azure/hooks/  
  - [ ] Fill the csv file and commit, the struct of the csv file can be found on [here](https://github.com/Azure/Edge-infrastructure-quickstart-template/blob/main/doc/Scale-with-automation.md#methodology)
  - [ ] Wait for the scale workflow run, and then resolve this comment and merge the pull request

 If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com